bin/rollback

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import os
+import boto3
+from src.get_current_live_environment import GetCurrentLiveEnvironment
+from src.flip_dns_record import FlipDNSRecord
+
+if __name__ == "__main__":
+    try:
+        aws_client = boto3.client('route53',
+                                  aws_access_key_id=os.getenv('AWS_ACCESS_KEY_ID'),
+                                  aws_secret_access_key=os.getenv('AWS_SECRET_ACCESS_KEY'),
+                                 )
+        current_live = GetCurrentLiveEnvironment(os.getenv('TARGET_DEPLOY_DOMAIN'), aws_client).call()
+        print(FlipDNSRecord(
+            client=aws_client,
+            live=current_live,
+            blue=os.getenv('BLUE_DEPLOY_DOMAIN'),
+            green=os.getenv('GREEN_DEPLOY_DOMAIN'),
+            target_domain = os.getenv('TARGET_DEPLOY_DOMAIN')
+        ).call()['ChangeInfo']['Comment'])
+
+    except RuntimeError as error:
+        print(format(error))
```

Comment:
Is there much difference in here compared to the flip command? Is there any way to reduce the duplication or just reuse the existing script?

Reply:
We're keeping the scripts self contained for now